andela-uenyingwa/dms-api
# #140287701 continuos integration

#### What does this PR do?
This PR integrates:

- Travis to test and build project for continuous integration.
- Coveralls coverage integration badge to the repository to display code coverage percentage.
- HoundCI to ensure adherence to coding style standards.
- Code Climate consolidates the results from a suite of static analysis tools into a single, real-time report, to improve code quality.
- Webpack a module bundler for modern JavaScript applications.

#### Description of Task to be completed?
- Integration of the Travis CI with the README badge.
- Integrate coveralls badge to show code coverage percentage.
- Create configuration file for HoundCI and integrate it to the repository.
- Integrate codeclimate badge to show code GPA.
- Integrate webpack and babel.


#### How should this be manually tested?
- Travis: Check to see the TRAVIS CI README badge on the github repository README.md.
- Coveralls: Open README.md file in the repository, a coverage badge display the percentage of the code coverage.
- HoundCi: You make a pull request and if there’s any error, HoundCI will make a comment on the pull request.
- Codeclimate: Open README.md file in the repository, a codeclimate badge display the GPA of the code. 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
140287701


#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
